.github/workflows/build-daily.yml

```diff
@@ -0,0 +1,85 @@
+name: Daily Build
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  Build:
+    name: ${{ matrix.os }} - ${{ matrix.configuration }} - Vulkan ${{ matrix.vulkan }}
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        configuration: [Debug, RelWithDebInfo]
+        os: [windows-2025, windows-2022, ubuntu-24.04, macos-15, macos-14, macos-13]
+        vulkan: [1.4.313.0, 1.3.296.0]
+
+    steps:
+      - name: Preinstall dependencies
+        if: ${{ runner.os == 'Linux' }}
+        run: |
+          sudo apt-get update
+          sudo apt-get install libglm-dev cmake libxcb-dri3-0 libxcb-present0 \
+            libpciaccess0 libpng-dev libxcb-keysyms1-dev libxcb-dri3-dev \
+            libx11-dev libwayland-dev libxrandr-dev libxcb-randr0-dev \
+            libxcb-ewmh-dev git python-is-python3 bison libx11-xcb-dev \
+            libxcursor-dev libxi-dev libxinerama-dev libxkbcommon-dev \
+            libxext-dev liblz4-dev libzstd-dev ocaml-core ninja-build \
+            pkg-config libxml2-dev wayland-protocols python3-jsonschema \
+            clang-format qtbase5-dev qt6-base-dev
```

Comment:
Do we need `wayland-scanner`?

Reply:
Most of these were copied from the Vulkan SDK instructions, but I do believe wayland-protocols is used by GLFW to compile Wayland support, so yes?